e2e-tests/native/wdio.conf.ts

```diff
@@ -0,0 +1,42 @@
+import {Capabilities} from '@wdio/types/build/Capabilities'
+
+const androidCapabilities: Capabilities = {
+    platformName: 'android',
+    'appium:app': '../native/android/app/build/outputs/apk/debug/app-debug.apk',
```

Comment:
Do we really want to use the app-debug here? Make release it more appropriate.

Reply:
This config should manly be used for local testing. Therefore, I think the better workflow is to develop the tests on the e2e debug apk locally and then let it execute the release config in the CI. Imo there is no difference between the release and the e2e build config (which skips the intro slides) here.